origin-dapp/src/components/conversation.js

```diff
@@ -233,78 +275,81 @@ class Conversation extends Component {
     }
   }
 
+  formatOfferMessage(info) {
+    const { listing = {} } = this.state
+    const { users, intl, smallScreenOrDevice, withListingSummary } = this.props
+
+    if (smallScreenOrDevice || !withListingSummary) return
+
+    const { returnValues = {}, event, timestamp } = info
+    const partyAddress = formattedAddress(returnValues.party)
+    const user = users.find((user) => formattedAddress(user.address) === partyAddress)
+    const userName = abbreviateName(user)
+    const party = userName || truncateAddress(returnValues.party)
+
+    const offerMessages = {
+      'OfferCreated': `${party} ${intl.formatMessage(
+        this.intlMessages.offerCreated
+      )}`,
+      'OfferWithdrawn': `${party} ${intl.formatMessage(
+        this.intlMessages.offerWithdrawn
```

Comment:
@Natasha08 these should be complete sentences, if possible. I know that we don't always do this elseqhere in the code, but we should ([like this](https://github.com/OriginProtocol/origin/commit/9ed6ead33672b2ea6c46b2b73e89926dcb54cf80#diff-61c348e20f66a533d512610bffbebad3R83)).